dask/dataframe/io/parquet/arrow.py

```diff
@@ -432,6 +475,7 @@ def write_partition(
                     metadata_collector=md_list,
                     **kwargs,
                 )
+            md_list[0].set_file_path(filename)
```

Comment:
Do we know that `md_list` has at least one element at this point?

Reply:
Good question - I added a check just in case something goes wrong.